redwood-layout-composeui/src/commonMain/kotlin/app/cash/redwood/layout/composeui/ComposeUiFlexContainer.kt

```diff
@@ -158,6 +160,7 @@ internal class ComposeUiFlexContainer(
         modifier = modifier.verticalScroll(rememberScrollState())
       }
     }
+    modifier = modifier.then(testOnlyModifier)
```

Comment:
I really dislike having test specific code in production code

Reply:
If you want to reduce its impact, using a nullable `Modifier` would actually be better. Most shrinking tools on any platform will see the value is never written and will inline the `null`. Then the `if` statement around it would become a constant and get completely eliminated.

But it's not a huge deal either way.

Reply:
Oooo good to know! Updated the PR using a nullable `Modifier`